airflow/decorators/base.py

```diff
@@ -101,22 +102,22 @@ class DecoratedOperator(BaseOperator):
     :type kwargs_to_upstream: dict
     """
 
-    template_fields = ('op_args', 'op_kwargs')
+    template_fields: Tuple[str, ...] = ('op_args', 'op_kwargs')
```

Comment:
We need to unify typing for this, otherwise it’d cause subclass problems when we annotate BaseOperator. I think `Sequence[str]` is probably better since both tuple and list are used pretty extensively in the code base?

cc @ashb since you’re working on models (which includes BaseOperator)